projects/solar/test/test_sense_temperature.c

```diff
@@ -12,6 +12,11 @@
 
 #define TEST_ADC_RAW_READING 0x800
 
+// The two test variables below are calculated from inputting 0x800 into the relative conversion
+// formulae (see sense_temperature.c)
+#define TEST_RTD_CONVERTED_READING -59
```

Comment:
Nitpick: it's probably a good idea to wrap this in parentheses like `(-59)` so that e.g. `foo TEST_RTD_CONVERTED_READING` isn't accidentally valid (expanding to `foo - 59`).